apple-pie/styles/style.css

```
@@ -0,0 +1,95 @@
+body {
+    display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: center;
+    align-items: center;
+}
+
+header{
+    
+ 
+    background-image: url(../images/apple-pie.jpg);
+ background-size: cover;
+ background-repeat: no-repeat;
+ 
+ color:white;
+ font-size: 100px;
+
+ width:1000px;
+ height:300px;
+ display:flex;
+ justify-content: center;
+ 
+
+}
+
+.recipe-story{
+    display:flex;
+    justify-content: center;
+
+
+}
+
+.bar {
+    width:50%;
+   display: flex;
+   justify-content: center;
+	align-items: center;
+}
+
+.bar1{
+    width: 50%;
+display: flex;
+    justify-content: center;
+	align-items: center;
+}
+
+
+#section1{
+    display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	align-items: center;
+    align-content: stretch;
+
+    
+    
+
+}
+.recipe-story{
+
+    width:40vw;
+}
+
+#section2{
+    display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	align-items: flex-start;
+    align-content: stretch;
+    width:600px;
+}
+
+
+
+#section3{
+    display: flex;
+	flex-direction: column;
+	flex-wrap: nowrap;
+	justify-content: flex-start;
+	align-items: flex-start;
+    align-content: stretch;
+    
+
+    width: 600px;
+}
+
+
+hr {
+    border: 1px solid rgba(0, 0, 0);
+    width:600px;}
```

the last } should be in the next line
```suggestion
      width:600px;
}
```